indexer-app/test/cmr/indexer/test/data/index_set_generics.clj

```diff
@@ -0,0 +1,48 @@
+(ns cmr.indexer.test.data.index-set-generics
+  (:require
+   [clojure.test :refer :all]
+   [cmr.indexer.data.index-set-generics :as gen]
+   [cmr.system-int-test.utils.dev-system-util :as dev-sys-util]))
+
+(def index-definition-def 
+  {:MetadataSpecification 
+   {:URL "https://cdn.earthdata.nasa.gov/generics/dataqualitysummary/v1.0.0"
+    :Name "DataQuailtySummary",
+    :Version "1.0.0"} 
+   :SubConceptType "DQS", 
+   :IndexSetup {:index {:number_of_shards 5, :number_of_replicas 1, :refresh_interval "1s"}},
+   :Indexes [{:Description "Name", :Field ".Name", :Name "Name", :Mapping "string"}
+             {:Description "Id - This is the old ECHO 10 GUID.", :Field ".Id", :Name "Id", :Mapping "string"}]})
+
+(deftest get-settings-test
+  "Test getting the settings from either the config file, the environment variable, or the default value." 
```

Comment:
`deftest` doesn't support docstrings like this. This should be a comment or a `testing` block

Reply:
I have put the text into a comment.